app/models/vendor_api_request.rb

```diff
@@ -1,3 +1,32 @@
 class VendorAPIRequest < ApplicationRecord
   belongs_to :provider, optional: true
+  scope :validation_errors, -> { where(status_code: 422) }
+
+  def self.list_of_distinct_errors_with_count
+    distinct_errors = validation_errors.flat_map do |request|
```

Comment:
Can this be called `error_messages` or similar? It looks like they aren't distinct until we tally them up